libs/build_dod_dataset.py

```diff
@@ -105,6 +112,120 @@ def get_projections_df():
     "Unassigned": NULL_VALUE
 }
 
+def get_state_and_counties():
+    # go to the JHUDataset to get this, for w/e reason
+    timeseries = JHUDataset.local().timeseries()
+    timeseries = timeseries.get_subset(
+        AggregationLevel.COUNTY,
+        after=datetime.datetime(2020, 3, 7),
+        country="USA", state=None
+    )
+
+    counties_by_state = defaultdict(list) # why tho
+
+    county_keys = timeseries.county_keys()
+    for country, state, county, fips in county_keys:
+        counties_by_state[state].append((county, fips))
+
+    return counties_by_state
+
+def find_peak(projection, col):
+    peak_row = []
+    for row in projection:
+        if not peak_row or (float(row[col]) > float(peak_row[col])):
+            peak_row = row
+    return peak_row
+
+def get_county_projections():
+    # for each state in our data look at the results we generated via run.py
+    # to create the projections
+    intervention_type = NO_INTERVENTION # None, as requested
+
+    # get 16 and 32 days out from now
+    today = datetime.datetime.now()
+    sixteen_days = today + datetime.timedelta(days=16)
+    thirty_two_days = today + datetime.timedelta(days=32)
+
+    #save results in a list of lists, converted to df later
+    results = []
+
+    # get the state and fips so we can get the files (TODO: this is trash)
+    missing = 0
+    for state, counties in get_state_and_counties().items():
+        for county, fips in counties:
+            file_name = f"{state}.{fips}.{intervention_type}.json"
+            path = os.path.join(OUTPUT_DIR_COUNTIES, file_name)
```

Comment:
Zero need to change, this but figured I'd throw it out there because I absolutely have come to adore [pathlib](https://docs.python.org/3/library/pathlib.html) (apart from urls...) and now have become its evangelist. the pathlib version of this would be:
```
import pathlib

path = pathlib.Path(OUTPUT_DIR_COUNTIES) / filename
if path.exists():
    with path.open("r) as projections:
```